pkgs/applications/networking/cluster/kubernetes/default.nix

```diff
@@ -41,39 +46,46 @@ stdenv.mkDerivation rec {
     patchShebangs ./hack
   '';
 
-  WHAT=concatStringsSep " " components;
+  WHAT = lib.concatStringsSep " " ([
+    "cmd/kubeadm"
+    "cmd/kubectl"
+  ] ++ components);
 
   postBuild = ''
     ./hack/update-generated-docs.sh
     (cd build/pause && cc pause.c -o pause)
   '';
 
   installPhase = ''
-    mkdir -p "$out/bin" "$out/share/bash-completion/completions" "$out/share/zsh/site-functions" "$man/share/man" "$pause/bin"
+    for p in $WHAT; do
```

Comment:
Can we use a different name instead of WHAT?

Reply:
`WHAT` is from the upstream makefile.